native/package.json

```diff
@@ -31,7 +31,7 @@
     "verbose": true,
     "testRegex": ".*-test(\\.ios)?(\\.android)?\\.js$",
     "transformIgnorePatterns": [
-      "node_modules/(?!react-native|bpk)"
+      "node_modules/(?!react-native|bpk|@skyscanner)"
```

Comment:
Because I have published our fork of `react-native-transitiongroup` under the `@skyscanner` scope, jest needs to know that it should run babel transforms on this module.